### ISSUE TITLE
SmartyPants should not convert text in a :command: role

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1496]
From my understanding of the [command](http://sphinx-doc.org/markup/inline.html#role-command) role, it would be appropriate to include _--option-flags_, like in the following example:

```
:command:`coverage run --source=myplugin.py test_myplugin.py`
```

With [html_use_smartypants](http://sphinx-doc.org/config.html?highlight=dash#confval-html_use_smartypants) on, these double hyphens get changed into a typographically correct entity, but in this case this is undesirable as it will give people an incorrect command when they copy&paste this into their terminal.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1496
- Originally reported by: [zoni](https://bitbucket.org/zoni)
- Originally created at: 2014-06-25T00:32:15.027

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-06-25 02:16:12+00:00_

Duplicate of #1359.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-06-25 02:17:59+00:00_

Thanks for reporting. The issue will be fixed at Sphinx-1.3 release.

**Comment by shimizukawa** [comment]
_From [zoni](https://bitbucket.org/zoni) on 2014-06-25 11:06:43+00:00_

Searching hadn't turned up #1359, doh. Sorry for the duplicate!
